index.js

```diff
@@ -17,7 +17,8 @@ class Listr {
 		this._RendererClass = CLIRenderer;
 		this._tasks = [];
 		this._options = Object.assign({
-			showSubtasks: true
+			showSubtasks: true,
+			parallel: false
```

Comment:
`parallel` => `concurrent`

http://stackoverflow.com/questions/1050222/concurrency-vs-parallelism-what-is-the-difference